### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by hitansivekariya33**

### CODE REVIEW

File: task5.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        for column in range(row, len):
+            sum_list = []
+            if (num_list[row] + num_list[column]) == sum_no:
+                sum_list.append(num_list[row])
+                sum_list.append(num_list[column])
+                main_list.append(sum_list)          
+num_list = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8]
+sum_no = int(input("Enter a sum::"))
+find(num_list, len(num_list), sum_no)
+tuple_list = list(set(tuple(sorted(sort_list)) for sort_list in main_list))
```
Comment (himanshu-suthar-tagline) [inline task5.py]: There are lot of conversion here. Would you please explain me what this is doing?

Comment (hitansivekariya33) [reply task5.py]: i used sorted in tuple  then convert tuple to set because remove duplicate tuple and then i covert set to list

Comment (himanshu-suthar-tagline) [reply task5.py]: That I can see, but why do you need this type of conversion?

Comment (hitansivekariya33) [reply task5.py]: Remove duplicate elements and convert list into list that's why i used

Comment (himanshu-suthar-tagline) [reply task5.py]: Please improve it, this is not the recommended way to do this.

**Pull request merged**